react/components/Pagination/index.js

```diff
@@ -160,6 +166,13 @@ Pagination.propTypes = {
   onPrevClick: PropTypes.func,
 
   hasPageTopIndicator: PropTypes.bool,
+
+  testIds: PropTypes.shape({
```

Comment:
Interesting. This brings a new pattern for test-id props within the Styleguide. I guess it makes sense for testing different elements within the component. What do you think of adding a section to the documentation to tell a bit about this?